webapp/src/features/adminEditOrganization/AdminEditOrganization.tsx

```diff
@@ -35,6 +47,23 @@ const AdminEditOrganization = () => {
     },
   })
 
+  const { orgName } = useParams<{ orgName: string }>()
+
+  useEffect(() => {
+    if (
+      (orgData ?? []).find((organization: AdminOrgSummary) => organization.name === orgName) &&
```

Comment:
Seeing an error on loading this page: 
![image](https://github.com/CDOT-CV/jpo-cvmanager/assets/2932159/73613487-fcfe-4d1c-ae75-b4b59e99aaef)

Reply:
this one is weird, there is a null organization?? This is inside the find method... Added the check, but still kinda confused

Reply:
This is resolved!